.github/workflows/sync.yml

```diff
@@ -0,0 +1,28 @@
+# The action is used to sync documentation of controls library with `kubescape.io` website
+# The action checks for any files that are out of sync 
+# And opens a pull request in the target repository with the changes(if any)
+
+name: Sync Files
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'docs/controls/**' # The action is triggered everytime there is a push to the defined path 
+  workflow_dispatch:
```

Comment:
do we want to keep the manual triggering?

Reply:
Technically, the files in the specified path are expected to only change when there's a release and that is when the action will be triggered. (push to specified folder typically means a modification to the controls library documentation, so needs to be synced to `Kubescape.io` website repo)